### DISCUSSION TITLE
Configure 5 rotation axes to control a robot arm.

### DISCUSSION THREAD

**Discussion by GrblGru** [grblHAL/core#324] [Q&A] [answered]
A GrblGru user wants to control his robot arm with GrblHAL.
For this purpose 0 linear axes and 5 rotational axes are needed.
Is it possible to configure this ?
Which inputs do I have to make in the Web Builder ?

**Comment by terjeio** [comment, marked as answer]
It is possible as long as running the controller in metric mode as there is an implied 1:1 relationship betwen mm and degrees. Designating an axis as rotary just disables the inches to mm transform.
A complicating factor might be the optional feed rate "fix", but I guess that does not apply - kinematics transform is handled by the CAM program (GrblGru)?

> Which inputs do I have to make in the Web Builder ?

Select a 5 axis build.

**Reply by GrblGru** [reply]
Thanks for the info.  I will try it when I am back from vacation. With the feed rates with robot arms I have unfortunately no experience. If problems should arise, one will also find a solution for it :)

Maybe another suggestion for the Web Builder. 
It would be very useful, if you could select the names of the axes explitit.

Thanks again for your great help.
